charts/kubeclarity/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if (((.Values.kubeclarity).prometheus).serviceMonitor).enabled -}}
```

Comment:
Please remove the unnecessary brackets.
Also please remove the unnecessary separator `---`.
What about checking if `.Values.kubeclarity.prometheus.enabled`?

Reply:
separator removed.

The brackets make it nil safe.